### PR TITLE
feat(inputSchema): mapping inputSchema from ctx to handler

### PR DESCRIPTION
**Pull request by aligg73**
See thread on teams for the rationale.
Only thing I wasn't sure of is to wrap it in ctx.is('json) or not...would break your tests if I would.

### CODE REVIEW

**Comment by aligg73** [comment]
...wait, I should bump up the version

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "middy-koa-wrapper",
-  "version": "1.1.5",
+  "version": "1.1.6",
```
Comment (bbeesley) [inline package.json]: can you remove this and let semantic release do its thing? this is a new feature not a bugfix so it should be a minor not a patch. Other than that, looks fine.

Comment (aligg73) [reply package.json]: yes, done.

**Pull request merged**